app/views/content/non-uk-teachers/train-to-teach-in-england-as-an-international-student.md

```diff
@@ -162,46 +166,77 @@ If you are not training to teach in these subjects, any support you get will dep
 
 ### 4. Search and apply for teacher training courses
 
-[Find postgraduate teacher training courses](https://www.gov.uk/find-postgraduate-teacher-training-courses) in England leading to [qualified teacher status (QTS)](https://www.gov.uk/guidance/qualified-teacher-status-qts). When you have chosen your course, you can <a href="https://www.gov.uk/apply-for-teacher-training">apply for teacher training</a>.
+You can train in a school or in a university setting, and study full-time (over a year) or part-time (over 2 or more years). Some courses combine [qualified teacher status (QTS)](https://www.gov.uk/guidance/qualified-teacher-status-qts) with a [postgraduate certificate in education (PGCE) or postgraduate diploma in education (PGDE)](/what-is-a-pgce). These are optional: you do not need a PGCE or PGDE to teach in England.
 
-#### First apply for your training place, then apply for your visa
+Teacher training courses can also be 'fee-paying' or 'salaried'.
 
-If you need a visa to train to teach in England, you must have a confirmed offer of a training place on a course offering student or skilled worker visa sponsorship before you apply for your visa.
+#### Fee-paying and salaried teacher training courses
 
-You can [search for courses where student or skilled worker visa sponsorship is available](https://www.gov.uk/find-postgraduate-teacher-training-courses). You can also search by location, teaching subject and age group, and for courses which pay a salary.
+A fee-paying teacher training course means you pay a fee to study as a teacher. Most teacher trainees in England are enrolled on this type of course.
 
-Due to rules about immigration, not all providers can accept international applicants – we suggest you contact them before you apply to check.
+Fee-paying courses include:
 
-<div class="inset">
-  <h4>Already living in the UK?</h4>
+* university teacher training programmes
+* school-centred initial teacher training (SCITT) programmes
+* School Direct unsalaried teacher training programmes
 
-  <p>
-If you have settled or pre-settled status under the <a href="https://www.gov.uk/settled-status-eu-citizens-families">EU Settlement Scheme</a>, or <a href="https://www.gov.uk/guidance/indefinite-leave-to-remain-in-the-uk">indefinite leave to remain</a>, you can apply to any English teacher training provider.
-  </p>
-</div>
+A salaried teacher training course means you are paid to work in a school while you learn to teach. However, these courses are limited in number and very competitive. 
+
+Salaried courses include:
+
+* School Direct salaried teacher training programmes
+* the Teach First Leadership Development programme
+* postgraduate teaching apprenticeships – [to apply for an apprenticeship, you'll need to meet certain visa requirements](/non-uk-teachers/visas-for-non-UK-trainees)
```

Comment:
```suggestion
* postgraduate teaching apprenticeships – to apply for an apprenticeship, you'll need to meet certain [visa requirements](/non-uk-teachers/visas-for-non-UK-trainees)
```

Reply:
Isn't the longer link more meaningful? Visa requirements is a bit generic?